demo/public/css/index.scss

```diff
@@ -350,3 +345,18 @@ body {
 }
 
 //Loader ends
+
+//slider starts
+.rc-slider-mark {
+    top: 14px;
+}
+
+.rc-slider-mark-text:first-child {
+    left: 3% !important;
+    width:40px;
+}
+.rc-slider-mark-text:last-child {
+    left: 97% !important;
```

Comment:
can we avoid `!important`?

Reply:
It's external component. I am trying to override an inline style.